test/check-application

```diff
@@ -1238,17 +1247,40 @@ class TestApplication(testlib.MachineCase):
         if auth or self.has_cgroupsV2:
             cpu = get_cpu_usage(IMG_BUSYBOX)
 
-            self.assertIn('/', memory)
-            numbers = memory.split('/')
-            self.assertTrue(numbers[0].strip().replace('.', '', 1).isdigit())
-            full = numbers[1].strip().split()
-            self.assertTrue(full[0].replace('.', '', 1).isdigit())
-            self.assertIn(full[1], ["GB", "MB"])
-
-            # Test that the value is updated dynamically
+            # swamped-crate has no explicit --memory limit
+            # FIXME: current format_bytes() does "KB"; fixed in https://github.com/cockpit-project/cockpit/pull/20356
```

Comment:
I believe this should now already be in `pkg/lib`, as I pulled and merged an update yesterday.

But feel free to do this in a follow up PR.

Reply:
Ah right -- I was concerned that this has to work against older cockpit-bridge, but podman bundles cockpit.js now. I'll send a follow-up!

Reply:
→ PR #1692